### PR TITLE
Remove (unused) foreign keys from api-db

### PR DESCRIPTION
**Pull request by CGoodwin90**
# Checklist

- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated
- [ ] PR title is ready for inclusion in changelog

A migration to remove all unused foreign key references from the database to allow for an easier transition to Knex migrations in a future release.

# Closing issues

N/A

### CODE REVIEW

File: services/api-db/docker-entrypoint-initdb.d/01-migrations.sql [hunk 1]
```diff
@@ -1772,6 +1772,706 @@ CREATE OR REPLACE PROCEDURE
...
+    IF EXISTS (
+      SELECT *
+      FROM INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS
+      WHERE
+        CONSTRAINT_SCHEMA = 'infrastructure'
+        AND REFERENCED_TABLE_NAME = 'user'
+        AND CONSTRAINT_NAME = 'user_ssh_key_ibfk_1'
+    ) THEN
+      ALTER TABLE `user_ssh_key`
+      MODIFY usid int, DROP FOREIGN KEY user_ssh_key_ibfk_1;
```
Comment (tobybellwood) [inline services/api-db/docker-entrypoint-initdb.d/01-migrations.sql]: the usid here is already a `char(36)` from an earlier migration

File: services/api-db/docker-entrypoint-initdb.d/01-migrations.sql [hunk 2]
```diff
@@ -1772,6 +1772,706 @@ CREATE OR REPLACE PROCEDURE
...
+    ELSEIF EXISTS (
+    	SELECT NULL
+		  FROM INFORMATION_SCHEMA.COLUMNS
+		  WHERE
+        table_name = 'user_ssh_key'
+        AND table_schema = 'infrastructure'
+        AND column_name = 'usid'
+    ) THEN
+      ALTER TABLE `user_ssh_key`
+      MODIFY usid int;
```
Comment (tobybellwood) [inline services/api-db/docker-entrypoint-initdb.d/01-migrations.sql]: the usid here is already a `char(36)` from an earlier migration

File: services/api-db/docker-entrypoint-initdb.d/01-migrations.sql (hunk 1 above)
Comment (CGoodwin90) [reply services/api-db/docker-entrypoint-initdb.d/01-migrations.sql]: Good catch! Updated now

File: services/api-db/docker-entrypoint-initdb.d/01-migrations.sql (hunk 2 above)
Comment (CGoodwin90) [reply services/api-db/docker-entrypoint-initdb.d/01-migrations.sql]: Good catch! Updated now

**Pull request merged**